src/js/actions/sampler.js

```diff
@@ -414,12 +414,17 @@ define(function (require, exports) {
         return uiUtil.hitTestLayers(doc.id, coords.x, coords.y)
             .bind(this)
             .then(function (hitLayerIDs) {
-                var hitLayerMap = new Set(hitLayerIDs.toJS()),
-                    clickedLeafLayers = layerTree.leaves.filter(function (layer) {
-                        return hitLayerMap.has(layer.id);
-                    }),
-                    topLayer = clickedLeafLayers.last(),
+                var topLayer,
                     selected = layerTree.selected;
+                
+                hitLayerIDs.forEach(function (layerID) {
+                    var layer = layerTree.byID(layerID);
+                    
+                    if (!layer.isArtboard) {
```

Comment:
`hitLayerIDs` already excludes group layers, so here we just need to make sure the layer is not an art board.